src/commons/assessmentWorkspace/__tests__/AssessmentWorkspace.tsx

```diff
@@ -1,17 +1,20 @@
-import { mount } from 'enzyme';
+import { render, screen } from '@testing-library/react';
+import { require as acequire } from 'ace-builds';
 import { Provider } from 'react-redux';
 import { createMemoryRouter, RouterProvider } from 'react-router';
 import { mockInitialStore } from 'src/commons/mocks/StoreMocks';
+import { renderTreeJson } from 'src/commons/utils/TestUtils';
 import { assertType } from 'src/commons/utils/TypeHelper';
+import { EditorBinding, WorkspaceSettingsContext } from 'src/commons/WorkspaceSettingsContext';
 
-import { EditorProps } from '../../editor/Editor';
 import { mockAssessments } from '../../mocks/AssessmentMocks';
 import AssessmentWorkspace, { AssessmentWorkspaceProps } from '../AssessmentWorkspace';
-const MockEditor = (props: EditorProps) => <div id="mock-editor">{props.editorValue}</div>;
-// mock editor for testing update
-jest.mock('../../editor/Editor', () => (props: EditorProps) => (
-  <MockEditor {...props}></MockEditor>
-));
+
+jest.mock('ace-builds', () => ({
+  ...jest.requireActual('ace-builds'),
+  require: jest.fn()
+}));
+const acequireMock = acequire as jest.Mock;
```

Comment:
Was this naming choice intentional, or a typo?

Reply:
intentional as per Editor.tsx, so as to not override the default `require` in JS